### ISSUE TITLE
Is there any way to recover deleted files on a DiskCryptor encrypted volume?

### ISSUE THREAD

**Issue by Beliathal** [DavidXanatos/DiskCryptor#72]
I've accidentally deleted a few files on a partition encrypted with DiskCryptor. The files were deleted with Total Commander (F8 bypasses Windows Recycle Bin). Any attempt at recovering these files with any recovery tool I have tried thus far, such as Handy Recovery, Recover My Files, Auslogics File Recovery, Active File Recovery etc - succeeded in recovering the files, but their contents remained encrypted and the files are unreadable. Is there any way to decrypt them or make DiskCryptor somehow export raw data (by start/end sectors) into an unencrypted file?

**Comment by DavidXanatos** [comment]
I think your best bet would be to 
1. make a full sector by sector backup, always habe a backup, always
2. decrypt the volume
3. run whatever recovery software you have on the decrypted volume.

**Comment by Beliathal** [comment]
Thank you for replying. I did exactly that. Made backups, made sure the data wasn't overwritten, decrypted the drive and tried to recover again. Sadly, it worked only partialy. I've managed to recover text files, but all binary data files seem to have their contents scrambled, if I load them in hex editor I don't see any usual file header info, just random code. Maybe some tool like OpenSSL could help decode them?
